packaging/aznfs/DEBIAN/postinst

```diff
@@ -37,4 +41,8 @@ set -e
 # Start aznfswatchdog service.
 systemctl daemon-reload
 systemctl enable aznfswatchdog
-systemctl start aznfswatchdog
\ No newline at end of file
+systemctl start aznfswatchdog
+
+# Start azfilenfs-watchdog service.
+systemctl enable azfilenfs-watchdog
```

Comment:
>azfilenfs-watchdog [](http://example.com/codeflow?start=17&length=18)

given that we start azfilenfs watchdog unconditionally, shall we have one watchdog instead?

Reply:
Now, we are having one watchdog.